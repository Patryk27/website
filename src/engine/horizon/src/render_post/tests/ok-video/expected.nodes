Elem(
    Spanned {
        inner: Elem {
            name: Spanned {
                inner: "video",
                span: Span {
                    start: 1,
                    end: 6,
                },
            },
            attrs: [
                Attr {
                    name: Spanned {
                        inner: "src",
                        span: Span {
                            start: 7,
                            end: 10,
                        },
                    },
                    value: Some(
                        Spanned {
                            inner: "https://foo/bar.jpg",
                            span: Span {
                                start: 11,
                                end: 32,
                            },
                        },
                    ),
                },
            ],
            children: [],
        },
        span: Span {
            start: 0,
            end: 35,
        },
    },
)
Text(
    Spanned {
        inner: "\n",
        span: Span {
            start: 35,
            end: 36,
        },
    },
)