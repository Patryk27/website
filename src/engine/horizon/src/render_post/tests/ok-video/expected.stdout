<video src="https://foo/bar.jpg" controls></video>
